gridstatus/spp.py

```diff
@@ -871,7 +878,13 @@ def _get_real_time_5_min_data(
 
         log(f"Getting data for {date} from {url}", verbose=verbose)
 
-        df = pd.read_csv(url)
+        try:
```

Comment:
why do we need this try except if we are calling `_format_5_min_url` above? isn't that supposed to handle this?

Reply:
Yeah, this isn't needed. It's leftover from a previous approach.